test/replication-luatest/gh_7318_box_wait_limbo_acked_test.lua

```diff
@@ -0,0 +1,525 @@
+local t = require('luatest')
```

Comment:
I didn't review the tests in too much detail because I assume it is expected to be merged before Friday to the release, no matter what I say. But I was able to leave a couple of comments which I believe could be useful to make the tests easier to comprehend. Right now they are all quite hard to read unless you are heavily in the context right now. Also due to relying on number of yields performed somewhere inside qsync code they look quite fragile.

See my other comment about the yield counts. And to the tests being hard to understand - it would help to explain in each test case in a comment what are you doing exactly and why. Otherwise from just the test case name it is much harder to find out than it should be.

Reply:
I've added more comments and also removed `server_wait_fiber_csw_ge_than()` where possible.